src/Platform.Renderers/src/Xamarin.Forms.Platform.Android/Extensions/DrawableExtensions.cs

```diff
@@ -1,23 +1,22 @@
 ﻿using System;
 using Android.Graphics;
+using Xamarin.Forms;
 using AColor = Android.Graphics.Color;
 using AColorFilter = Android.Graphics.ColorFilter;
 using ADrawable = Android.Graphics.Drawables.Drawable;
+
+#if __ANDROID_29__
 using ADrawableCompat = AndroidX.Core.Graphics.Drawable.DrawableCompat;
+#else
+using ADrawableCompat = Android.Support.V4.Graphics.Drawable.DrawableCompat;
+#endif
 
-namespace Xamarin.Forms.Platform.Android
+namespace Xamarin.Platform
```

Comment:
If this is Xamarin.Platform shouldn't this move to Platform.Handlers folder =?

Reply:
`DrawableExtensions.cs` have been moved to Platform.Handlers.